DesignPatternExamples/Assets/Design Patterns/Builder/Scripts/IAirshipBuilder.cs

```diff
@@ -0,0 +1,15 @@
+using System.Collections.Generic;
+
+namespace Builder
+{
+    public interface IAirshipBuilder
+    {
+        List<IAirshipModule> airshipModules { get; set; }
+        void SetLeftWing(bool value);
+        void SetRightWing(bool value);
+        void SetLeftWeapon(bool value);
+        void SetRightWeapon(bool value);
+        void SetFrontalWeapon(bool value);
+        void GetAirship();
```

Comment:
It's strange that there's a `Get` method returning void, consider changing this name